docs/UserGuide.md

```diff
@@ -676,7 +676,7 @@ Shows a message explaining how to access the User Guide or obtain in-app guidanc
 
 ### View in-app guidance: `help COMMAND` <a name="help_command"></a>
 
-Displays concise instructions for users to learn the given `COMMAND`'s function.
+This command displays concise instructions for users to learn the given `COMMAND`'s function.
```

Comment:
Change to: "This command displays concise instructions within the app for you to learn the given `COMMAND`'s function."